src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -184,9 +184,12 @@ class BasicSettings extends React.Component {
     const okProvisionTemplate = dataSet.provisionSource === 'template' &&
         [ null, dataSet.clusterId ].includes(templates.getIn([ dataSet.templateId, 'clusterId' ]))
 
+    const clusterArch = clusters.getIn([ dataSet.clusterId, 'architecture' ]) || '_'
```

Comment:
`_` is used in several places as default architecture. Consider using a constant with a self-explanatory content.

Reply:
Right - I'll add the constant.